command/server/config_test.go

```diff
@@ -57,3 +57,7 @@ func TestConfigRaftRetryJoin(t *testing.T) {
 func TestParseSeals(t *testing.T) {
 	testParseSeals(t)
 }
+
+func TestUnknownFieldValidation(t *testing.T) {
```

Comment:
This is great! Can we also have a test that checks the filename for unusedKeys is correct, for configs between multiple files?